packages/fxa-auth-server/lib/senders/emails/layouts/subscription/en.ftl

```diff
@@ -16,5 +17,9 @@ subplat-cancel-plaintext = { subplat-cancel }:
 subplat-reactivate = Reactivate subscription
 subplat-reactivate-plaintext = { subplat-reactivate }:
 subplat-update-billing = Update billing information
+subplat-privacy-policy = { -brand-mozilla } Privacy Policy
+subplat-privacy-policy-plaintext = { subplat-privacy-policy }:
+subplat-cloud-terms = { -brand-firefox } Cloud Terms of Service
```

Comment:
Oh I'm sorry I missed this in my first review. We currently have "Firefox Cloud" as a brand now. This should be `{ -brand-firefox-cloud } Terms of Service`

Reply:
omg I'm ashamed for not realizing Cloud Terms of Service is not a thing. 😂